main.py

```diff
@@ -34,7 +34,10 @@
     "setupverification": setup_verification,
     "getrandomquote": get_random_quote,
     "kavirgoat": kavir_goat,
-    "purge" : purge
+    "purge" : purge,
+    "GitHub" : show_github,
```

Comment:
Commands are not case sensitive, it is unnecessary to have this here three times.